kit/aws/landingzones/cloud-native/documentation.tf

```diff
@@ -0,0 +1,10 @@
+output "documentation_md" {
+  value = <<EOF
+# Cloud Native Landing Zone
+
+This landing zone is inteded for applications teams that want to work natively without AWS and can take
```

Comment:
q: what does it mean without AWS in this context?

Reply:
i dont know :), I copied it from meshcloud-dev...maybe it should mean "with" and not "without"...